src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+
+public class Block {
+	
+	public static final String EXAMPLE = "123";
+	private int blockNum;	
+	
+	// No validation check need as constructor only accepts integer as block number
+	public Block(int blockNumber) {
+		this.blockNum = blockNumber;
+	}
+
+	/**
+	 * @return the blockNum
```

Comment:
We need to have more meaningful header comments. For a getter its ok to not have a comment. If you are adding comments make sure its needed and meaningful.